lib/power-bi/user.rb

```diff
@@ -0,0 +1,51 @@
+module PowerBI
+  class User
+    attr_reader :email_address, :group_user_access_right, :display_name, :identifier, :principal_type
+
+    def initialize(tenant, data)
+      @email_address = data[:emailAddress]
+      @group_user_access_right = data[:groupUserAccessRight]
+      @display_name = data[:displayName]
+      @identifier = data[:identifier]
+      @principal_type = data[:principalType]
+      @workspace = data[:workspace]
+      @tenant = tenant
+    end
+
+    def delete_user
+      @tenant.delete("/groups/#{@workspace.id}/users/#{@email_address}")
+      @workspace.users.reload
+    end
+
+  end
+
+  class UserArray < Array
+
+    def initialize(tenant, workspace)
+      super(tenant)
+      @workspace = workspace
+    end
+
+    def self.get_class
+      User
+    end
+
+    def create(email_address, identifier, group_user_access_right, display_name, principal_type)
```

Comment:
Isn't `email_address` and `group_user_access_right` enough?

Additionally, add `Viewer` as default for `group_user_access_right` (in case it is not specified)